backend/src/services/db.js

```diff
@@ -0,0 +1,61 @@
+require("dotenv").config();
+const { MongoClient } = require("mongodb");
+
+const uri = `mongodb+srv://${process.env.DB_USER}:${process.env.DB_PASSWORD}@cluster0.rstre.mongodb.net/${process.env.DB_NAME}?retryWrites=true&w=majority`;
```

Comment:
Sugestão criar um variável de ambiente no arquivo **.env** no o seguinte nome **DB_URL** , pois a url de conexão do banco pode mudar quando a aplicação for para ambiente de teste ou produção. Dessa forma fica mais simples alterar a informação quando for colocar a aplicação para rodar nós ambientes de teste ou produção.

Reply:
Boa sugestão, já implementei a mesma.